internal/apiutil/transport.go

```diff
@@ -18,35 +18,36 @@ import (
 // LoggingErrorEncoder is a go-kit error encoder logging decorator.
 func LoggingErrorEncoder(logger logger.Logger, enc kithttp.ErrorEncoder) kithttp.ErrorEncoder {
 	return func(ctx context.Context, err error, w http.ResponseWriter) {
-		switch err {
-		case ErrBearerToken,
-			ErrMissingID,
-			ErrBearerKey,
-			ErrInvalidAuthKey,
-			ErrInvalidIDFormat,
-			ErrNameSize,
-			ErrLimitSize,
-			ErrOffsetSize,
-			ErrInvalidOrder,
-			ErrInvalidDirection,
-			ErrEmptyList,
-			ErrMalformedPolicy,
-			ErrMissingPolicySub,
-			ErrMissingPolicyObj,
-			ErrMissingPolicyAct,
-			ErrMissingCertData,
-			ErrInvalidTopic,
-			ErrInvalidContact,
-			ErrMissingEmail,
-			ErrMissingHost,
-			ErrMissingPass,
-			ErrMissingConfPass,
-			ErrInvalidResetPass,
-			ErrInvalidComparator,
-			ErrMissingMemberType,
-			ErrInvalidAPIKey,
-			ErrMaxLevelExceeded,
-			ErrBootstrapState:
+		switch {
+		case errors.Contains(err, ErrBearerToken),
+			errors.Contains(err, ErrMissingID),
+			errors.Contains(err, ErrBearerKey),
+			errors.Contains(err, ErrInvalidAuthKey),
+			errors.Contains(err, ErrInvalidIDFormat),
+			errors.Contains(err, ErrNameSize),
+			errors.Contains(err, ErrLimitSize),
+			errors.Contains(err, ErrOffsetSize),
+			errors.Contains(err, ErrInvalidOrder),
+			errors.Contains(err, ErrInvalidDirection),
+			errors.Contains(err, ErrEmptyList),
+			errors.Contains(err, ErrMalformedPolicy),
+			errors.Contains(err, ErrMissingPolicySub),
+			errors.Contains(err, ErrMissingPolicyObj),
+			errors.Contains(err, ErrMissingPolicyAct),
+			errors.Contains(err, ErrMissingCertData),
+			errors.Contains(err, ErrInvalidTopic),
+			errors.Contains(err, ErrInvalidContact),
+			errors.Contains(err, ErrMissingEmail),
+			errors.Contains(err, ErrMissingHost),
+			errors.Contains(err, ErrMissingPass),
+			errors.Contains(err, ErrMissingConfPass),
+			errors.Contains(err, ErrInvalidResetPass),
+			errors.Contains(err, ErrInvalidComparator),
+			errors.Contains(err, ErrMissingMemberType),
+			errors.Contains(err, ErrInvalidAPIKey),
+			errors.Contains(err, ErrMaxLevelExceeded),
+			errors.Contains(err, ErrBootstrapState),
+			errors.Contains(err, errors.ErrInvalidQueryParams):
```

Comment:
You can probably do the same with `ErrUnsupportedContentType` and move both errors in `apiutil`

Reply:
On it @manuio!